consensus/scp/src/node.rs

```diff
@@ -138,8 +128,8 @@ pub trait ScpNode<V: Value>: Send {
     /// Get local node quorum set.
     fn quorum_set(&self) -> QuorumSet;
 
-    /// Submit a list of values of nomination.
-    fn nominate(
```

Comment:
This was renamed to more closely match Slot.propose_values, and to more clearly indicate that a proposed value does not necessarily get nominated immediately.